_posts/2015-02-04-omero-bio-formats-5-0-7.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  Release of OMERO & Bio-Formats 5.0.7
+intro-blurb: The OME Consortium is pleased to announce the release of OMERO & Bio-Formats 5.0.7
+---
+
+Today we are launching 5.0.7, a bug-fixing release.
+
+For OMERO this includes:
+
+    * Rendering improvements including 32 bit and float support
+    * Vast improvements in Mac launching (separate clients for your Java version)
+    * Faster import of complex plates
+    * OMERO.dropbox improvements
+    * ROI and measurement tool fixes
+    * OMERO.matlab updates
+
+Bio-Formats fixes include:
+
+    * ND filter parsing for DeltaVision
+    * Timepoint count and original metadata parsing for Metamorph
+    * Build issues when Genshi or Git are missing
+    * LZW image decoding
+    * Minor core formats (OME/XML/TIFF) improvements
+    * Minor doc improvements - Eclipse build instructions and documenting using Bio-Formats in Python (extensive developer doc improvements coming in 5.1)
+
+The software is available from [downloads.openmicroscopy.org/omero/5.0.7/](http://downloads.openmicroscopy.org/omero/5.0.7/) and [downloads.openmicroscopy.org/bio-formats/5.0.7/](http://downloads.openmicroscopy.org/bio-formats/5.0.7/)
+
+For information on upgrading your server, see the upgrade guide for system administrators - [server upgrade page](http://www.openmicroscopy.org/site/support/omero5/sysadmins/server-upgrade.html)
+
+For full details of the next major release, see the 5.1 milestone page - [trac.openmicroscopy.org.uk/ome/milestone/5.1.0](http://trac.openmicroscopy.org.uk/ome/milestone/5.1.0)
+
+Kind regards, The OME team
```

Comment:
@mtbc - which bit should I remove?  The whole section highlighted or the Kind reards, The OME team bit?